ui/tests/acceptance/settings/auth/configure/section-test.js

```diff
@@ -37,3 +42,16 @@ test('it can save options', function(assert) {
     );
   });
 });
+
+['aws', 'azure', 'gcp', 'github', 'kubernetes', 'ldap', 'okta', 'radius'].forEach(function(type) {
+  test(`it shows tabs for auth method: ${type}`, assert => {
+    let path = `${type}-${Date.now()}`;
+    cli.consoleInput(`write sys/auth/${path} type=${type}`);
+    cli.enter();
+    indexPage.visit({ path });
+    andThen(() => {
+      let numTabs = type === 'aws' ? 4 : 2;
```

Comment:
Looking at the data structure in the helper, it took me a few minutes to figure out why this wasn't `? 3 : 1` instead: https://github.com/hashicorp/vault/blob/9b91394a68a8ba774d9ea2bb3a34be70fd25d1d1/ui/app/helpers/tabs-for-auth-section.js#L67-L74

Maybe a comment in the test explaining would help clarify?

Reply:
Added, thanks!